website/pages/docs/from-1.5/special-variables.mdx

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+page_title: Special Variables - HCL Configuration Language
+sidebar_title: Special Variables
+description: |-
+  Special variables provide connection information and basic instance state information.
+  This page covers all existing special variables.
+---
+
+# Build Variables
+
+`@include 'from-1.5/beta-hcl2-note.mdx'`
+
+Build variables will allow you to access, from provisioners and post-processors, connection information and basic instance state information.
+All special build variables are prefixed with `build`:
+
+```hcl
+  provisioner "shell" {
+      environment_vars = ["TESTVAR=${build.PackerRunUUID}"]
+      inline = ["echo $TESTVAR"]
+  }
+```
+
+The valid build variables to request are:
+
+- **ID**: Represents the vm being provisioned. For example, in Amazon it is the instance id; in digitalocean,
+  it is the droplet id; in Vmware, it is the vm name.
+
+- **Host**, **Port**, **User** and **Password**: The host, port, user, and password that Packer uses to access the machine.
+  Useful for using the shell local provisioner to run Ansible or Inspec against the provisioned instance.
+
+- **ConnType**: Type of communicator being used. For example, for SSH communicator this will be "ssh".
+
+- **PackerRunUUID**: Current build's unique id. Can be used to specify build artifacts.
```

Comment:
`Can be used to specify build artifacts.` I think it isn't clear what that means, do you think it would be a good idea to add an example here ?